### DISCUSSION TITLE
Can't reapply for GitHub Students as the page is only reloading

### DISCUSSION THREAD

**Discussion by ItIzYe** [community/community#113529] [GitHub Education]
### Hi everyone,

Hi everyone, 
so the problem is basically already in the title. I'm trying to reapply myself for GitHub Students but everytime I submit the form, the page just reloads and I have to submit everything again. 
Does anybody know how to solve the problem?

**Comment by SebastienXia** [comment]
## Hi, @ItIzYe 

Did you mean, when you tried to upload an image ? (last step of the process)

**Reply by ItIzYe** [reply]
Exactely, that's the only thing I have to do when reapplying as all my other data (name, school mail etc.) is filled in automatically

**Reply by SebastienXia** [reply]
😁 Look at this link dude : https://github.com/orgs/community/discussions/113240#discussioncomment-8845926

`[close the thread and mark as answered if this helped you resolve your issue]`
